test/integration/setup-gitlab/bootstrap_gitlab_environment_test.go

```
@@ -38,37 +38,56 @@ func readLogsFromVm(t *testing.T, instanceName string, instanceZone string, inst
 	return shell.RunCommandAndGetStdOutE(t, gcloudCmd)
 }
 
-func TestBootstrapGitlabVM(t *testing.T) {
+func TestValidateStartupScript(t *testing.T) {
 	// Retrieve output values from test setup
 	setup := tft.NewTFBlueprintTest(t,
 		tft.WithTFDir("../../setup"),
 	)
-	url := setup.GetStringOutput("gitlab_url")
-	gitlabSecretProject := setup.GetStringOutput("gitlab_secret_project")
-	gitlabSecretProjectNumber := setup.GetStringOutput("gitlab_project_number")
-	gitlabPersonalTokenSecretName := setup.GetStringOutput("gitlab_pat_secret_name")
-	gitlabWebhookSecretId := setup.GetStringOutput("gitlab_webhook_secret_id")
 	instanceName := setup.GetStringOutput("gitlab_instance_name")
 	instanceZone := setup.GetStringOutput("gitlab_instance_zone")
-	gitlabTokenSecretId := fmt.Sprintf("projects/%s/secrets/%s", gitlabSecretProjectNumber, gitlabPersonalTokenSecretName)
-
+	gitlabSecretProject := setup.GetStringOutput("gitlab_secret_project")
 	// Periodically read logs from startup script running on the VM instance
-	for count := 0; count < 10; count++ {
+	for count := 0; count < 100; count++ {
```

Probably want to raise an error if not successful after the final iteration.

---

@caetano-colin - can include in a future PR